tests/go/GofmtBearTest.py

```diff
@@ -1,7 +1,15 @@
 from bears.go.GofmtBear import GofmtBear
 from coalib.testing.LocalBearTestHelper import verify_local_bear
 
-GofmtBear = verify_local_bear(
+GofmtBearTest = verify_local_bear(
     GofmtBear,
     ('package main\n\nfunc main() {\n\treturn 1\n}',),
     ('package main\nfunc main() {\n\treturn 1\n}',))
+
+GofmtBearWithSettingsTest = verify_local_bear(
```

Comment:
Is there an issue present on building a check method in LocalBearTestHelper?
For now, I will just use the `check_result` and a `LocalBearTestHelper` derived class to check for changes like some other tests have done.